src/NUnitEngine/nunit-agent/Program.cs

```diff
@@ -153,16 +153,6 @@ public static int Main(string[] args)
                 log.Error("Exception in RemoteTestAgent", ex);
             }
 
-            try
-            {
-                // Unregister the channel
-                testAgencyServer.Stop();
```

Comment:
Actually, the previous `ServerBase.Stop()` did more than just unregister the TCP channel, including stuff with @jnm2's fix for the agent crashes recently. Is none of this necessary in this case?

@jnm2 - you have a much deeper understanding of this process than I do, would you mind reviewing this one as well?